src/components/ButtonPanel.js

```diff
@@ -1,12 +1,39 @@
 import Button from './Button';
-import { data } from './Data';
+
+const group1 = ['AC', '+/-', '%', ' ÷'];
```

Comment:
**[Optional]**
I think these multiple declaration can be handle with a hash table, for better memory management. Actually this whole component can be refactored to contain a single button component and a single data for button groups.